Fixed incorrect UpdateField.h value BANKBAG_SLOT_LAST

HelloKitty opened the pull request:
I think this value is wrong?

PLAYER_FIELD_BANKBAG_SLOT_1 = 0x1a8 + UNIT_END, // Size:12
PLAYER_FIELD_BANKBAG_SLOT_LAST= 0xab2 + UNIT_END,

I think the new value is correct. There are no references to this value in Mangos I believe though so it's just a latent fault. Not an active one thankfully.

I also found this was wrong in CMangos and have made a PR there too: https://github.com/cmangos/mangos-classic/pull/339

HelloKitty committed:
Fixed incorrect UpdateField.h value BANKBAG_SLOT_LAST

I think this value is wrong?

PLAYER_FIELD_BANKBAG_SLOT_1 = 0x1a8 + UNIT_END, // Size:12
PLAYER_FIELD_BANKBAG_SLOT_LAST= 0xab2 + UNIT_END,

I think the new value is correct. There are no references to this value in Cmangos though so it's just a latent fault. Not an active one thankfully.

I also found this was wrong in CMangos and have made a PR there too: https://github.com/cmangos/mangos-classic/pull/339

HelloKitty commented:
Travis failed to compile 1/4 but shouldn't be a result of this PR.

MadMaxMangos commented:
:white_check_mark: [Build Mangos ZERO Server 21.14.1053 completed](https://ci.appveyor.com/project/MaNGOS/server-9fytl/build/21.14.1053) (commit https://github.com/mangoszero/server/commit/9fc778073e by @HelloKitty)

HelloKitty commented:
0x1b2 may be more appropriate. I don't know the size of the update field, 3.3.5 on Trinitycore says it is 2 indices.

Either way the current value is totally off, so either 0x1b2 or 0x1b3.

HelloKitty committed:
Accounted for potential 2 index size BANKBAG_SLOT in UpdateField

0x1b2 may be more appropriate. I don't know the size of the update field, 3.3.5 on Trinitycore says it is 2 indices so it might be 2 indices on 1.12.1 too.

MadMaxMangos commented:
:white_check_mark: [Build Mangos ZERO Server 21.14.1054 completed](https://ci.appveyor.com/project/MaNGOS/server-9fytl/build/21.14.1054) (commit https://github.com/mangoszero/server/commit/d9a11bbb2e by @HelloKitty)
